_posts/2020-04-11-native-mobile-to-flutter.markdown

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "Looking Back on Our Migration from Native Mobile to Flutter."
+date: 2020-04-11
+categories: [mobile, flutter]
+author: Bobby Manuel 
+---
+
+### Timeline of the project.
+
+- Project spanned Aug 2019 - Dec 2019 
+- Initial release to Android users in February 2020.
+- Release to all users Android + iOS March 2020.
+- As of the time of this article we've been live across all of our users for about a month with no Flutter related issues.
+
+
+### Why did we do it?
+
+We had two working native mobile applications, but lacked feature parity for Android and iOS users.
+
+Android and iOS development took place at different paces, with varying quality and cost.
+
+Maintaining redundancy across two mobile teams required us to maintain a minimum of four developers on mobile rather than as few as two in a cross platform environment.
+
+### We wanted to achieve...
+
+- feature parity on both platforms.
+- a unified mobile team with higher quality peer reviews. 
```

Comment:
Love the goal but wondering about the mechanism. Is it that review quality would be higher because of more eyes on each PR due to not having siloed projects?

Reply:
Exactly!

Reply:
I think this is a good example that these short bullet points cover the highlights but don't always explain the "why".

It might be good to use bullet points + explanation format, such as:

* *scalability and redundancy for our mobile team* - since everybody uses a common language and framework (Dart + Flutter)